haproxy/README.md

```diff
@@ -35,6 +37,16 @@ The Agent collects metrics via a stats endpoint:
 
 2. [Restart HAProxy to enable the stats endpoint][3].
 
+##### Versions >= 2
+
+The check supports a newer implementation starting with HAProxy version 2 that is based on a Prometheus endpoint:
+
+1. Configure your `haproxy.conf` using the [official guide][16].
+
+2. [Enable](#configuration) the setting `use_preview`.
```

Comment:
Can we be more explicit and have the config be `use_prometheus`? `preview` is quickly going to get outdated